scripts/detect_bad_deps.js

```diff
@@ -41,7 +41,7 @@ exec("flow check --profile", function(error, stdout, stderr) {
     process.exit(1);
   }
   console.log("Biggest cycle: " + cycle_len);
-  let MAX_CYCLE_LEN = 59;
+  let MAX_CYCLE_LEN = 60;
```

Comment:
Why this is needed? What import caused the cycle increase?

Reply:
Importing the realm in the debugger causes this. Even with `import type` or `import typeof` this cycle length increases by 1.

Reply:
Then, consider pass the necessary data into debugger instead of the whole realm object if possible. Increase the flow cycle increases the flow checking time, it is already very bad now....

Reply:
I have tried this before, but most objects needed (ExecutionContext, LexicalEnvironment, etc) have pointers to the realm, so importing any of them will cause this cycle increase. I checked with @cblappert before, there doesn't seem to be a good way to avoid this.